models/resnet.py

```diff
@@ -175,6 +239,14 @@ def _resnet(arch, Block, depth, pretrained):
     return model
 
 
+def resnet18(pretrained=False):
```

Comment:
Also need to set doc for ResNet18/34/...

Reply:
done, thanks.